examples/main.tf

```diff
@@ -0,0 +1,45 @@
+data "aws_iam_policy_document" "read-a-bucket" {
+  statement {
+    sid    = "AllowReadOfABucket"
+    effect = "Allow"
+
+    actions = [
+      "s3:GetObject",
+    ]
+
+    resources = [
+      "arn:aws:s3:::public-bucket/*",
+    ]
+  }
+}
+
+module "periodic_worker" {
+  source = "../"
+
+  lambda_code_bucket = "tools-infra-lambda-bucket"
+
+  lambda_code_path = "dummy.zip"
+
+  lambda_name = "alambda"
+
+  lambda_runtime = "nodejs6.10"
+
+  lambda_handler = "lib.default"
+
+  lambda_memory_size = "128"
+
+  lambda_timeout = "300"
+
+  tags = {
```

Comment:
Based on conventions, at minimum here is the tags for all resources:
Name, Environment, ProductDomain, Service, Description

Should we make it tied to the module instead?
Though, we might still need _tags_ variable to be used by another use case, i.e: dev might have different tagging policy, which can be applied use _tags_ variable.

Reply:
@rizkidoank what does service should be for the lambda?

When I look at https://29022131.atlassian.net/wiki/spaces/SI/pages/196952337/AWS+Resource+Tagging+Policy, there's no standard yet for lambda. Name, environment, product domain and description make sense. Not sure about service though.